analyzers/its/sources/CSharpLatest/CSharpLatest/CSharp9/GetWeatherForecastWithRecords.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Json;
+
+namespace CSharpLatest.CSharp9
+{
+    internal class GetWeatherForecastWithRecords
```

Comment:
Files in this directory are moved from `Net5`.
If it looks like a git `move`, it's a move.
If it looks like a git `add`, it's move from `Net5Samples` and a wrap inside a class.
`Net5Samples` was not a project, it was 14 projects (hide-the-pain) that had a single top-level `Program.cs` with only language features. 
So basically everything is a move, just slightly more elaborate.

Reply:
Thanks for further splitting your changeset into multiple dedicated commits.
That made the review much easier!